bindings/go/main.go

```diff
@@ -109,13 +107,31 @@ func (b *Blob) UnmarshalText(input []byte) error {
 	if err != nil {
 		return err
 	}
-	if len(bytes) != len(b) {
+	if len(bytes) != GetBytesPerBlob() {
 		return ErrBadArgs
 	}
-	copy(b[:], bytes)
+	*b = bytes
```

Comment:
The copy pattern does not work anymore? Would `copy(*b, bytes)` work? Is it the same as what you did?

Reply:
Here is how a similar function in geth is handled: https://github.com/ethereum/go-ethereum/blob/master/common/hexutil/json.go#L56

Perhaps we can replicate the geth logic and trust in their authority?

Reply:
That's pretty similar to how we do it now, but less straight-forward IMO. They use `hex.Decode` instead of `hex.DecodeString` and they have a `checkText` function. I would prefer to keep what we have.

Reply:
Thanks. Current approach looks fine!